india_compliance/gst_india/report/gstr_1_document_issued_summary/gstr_1_document_issued_summary.py

```diff
@@ -0,0 +1,196 @@
+            "label": _("Nature of Document"),
+            "fieldtype": "Data",
+            "width": 300,
+        },
+        {
+            "fieldname": "naming_series",
+            "label": _("Series"),
+            "fieldtype": "Data",
+            "width": 150,
+        },
+        {
+            "fieldname": "from_serial_no",
+            "label": _("Serial Number From"),
+            "fieldtype": "Data",
+            "width": 160,
+        },
+        {
+            "fieldname": "to_serial_no",
+            "label": _("Serial Number To"),
+            "fieldtype": "Data",
+            "width": 160,
+        },
+        {
+            "fieldname": "total_number",
+            "label": _("Total Submitted Number"),
+            "fieldtype": "Int",
+            "width": 180,
+        },
+        {
+            "fieldname": "canceled",
+            "label": _("Canceled Number"),
+            "fieldtype": "Int",
+            "width": 160,
+        },
+        {
+            "fieldname": "total_issued",
+            "label": _("Total Issued Documents"),
+            "fieldtype": "Int",
+            "width": 150,
+        },
+    ]
+
+
+def get_data(filters) -> list:
+    data = []
+
+    document_mapper = {
+        "Invoices for outward supply": "Sales Invoice",
+        "Invoices for inward supply": "Purchase Invoice",
+        "Invoices for inward supply from unregistered person": "Purchase Invoice",
+        "Debit Note": "Purchase Invoice",
+        "Credit Note": "Sales Invoice",
+        "Receipt Voucher": "Payment Entry",
+        "Payment Voucher": "Payment Entry",
+        "Receipt Voucher (JV)": "Journal Entry",
+        "Payment Voucher (JV)": "Journal Entry",
+    }
+
+    for nature_of_document, document_type in document_mapper.items():
+        data.extend(get_document_summary(filters, document_type, nature_of_document))
+
+    return data
+
+
+def get_document_summary(filters, document_type, nature_of_document):
+    doctype = frappe.qb.DocType(document_type)
+
+    query = (
+        frappe.qb.from_(doctype)
+        .select(
+            doctype.name, doctype.naming_series, doctype.creation, doctype.docstatus
+        )
+        .where(doctype.company == filters.company)
+        .where(doctype.posting_date.between(filters.from_date, filters.to_date))
+        .where(doctype.naming_series.isnotnull())
+        .where(doctype.docstatus > 0)
+    )
+
+    query = apply_document_wise_condition(filters, query, nature_of_document, doctype)
+
+    data = query.run(as_dict=True)
+
+    naming_series_data = {}
+
+    for item in data:
+        naming_series = parse_naming_series(
+            item.naming_series.replace("#", ""), doc=item
+        )
+
+        naming_series_data.setdefault(
+            naming_series, {"canceled_count": 0, "total_count": 0, "document_names": {}}
+        )
+
+        names = naming_series_data.get(naming_series)
+        if item.docstatus == 2:
+            names["canceled_count"] += 1
```

Comment:
If there is an amendment created from the Sales Invoice, the cancelled count should not be affected.